tests/cbmc/stubs/memset.c

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
these standard stubs should be moved to the starter kit.

Reply:
Yes, I created the issue https://github.com/awslabs/aws-templates-for-cbmc-proofs/issues/21 on the started kit repo to track this request.